api/user.yaml

```diff
@@ -212,6 +212,67 @@ paths:
         "500":
           description: Internal server error
 
+  /users/me:
+    get:
+      tags:
+        - users
+      summary: Get self user profile
+      security:
+        - OAuth2: [self]
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+              example:
+                id: "e1b6ba6c-deb2-4d24-87b0-bbf79fa4487c"
+                organizationUnit: "26eec421-f1bb-4deb-a5d3-9ab6554c2ae6"
+                type: "employee"
+                attributes:
+                  username: "alice.wu"
+                  firstname: "Alice"
+                  lastname: "Wu"
+                  email: "alice.wu@company.inc"
+        "401":
+          description: Unauthorized - missing or invalid token
+        "404":
+          description: Authenticated user not found
+        "500":
+          description: Internal server error
+    patch:
+      tags:
+        - users
+      summary: Update specified attributes of the self profile
+      security:
+        - OAuth2: [self]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateCurrentUserRequest'
+            example:
+              attributes:
+                firstname: "Alice"
+                lastname: "Wu"
+                password: "n3wP@ssword!"
+      responses:
+        "200":
+          description: Updated profile of the current user
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
```

Comment:
Shall we update the example here? It's showing empty attributes in the swagger editor